spyder/plugins/variableexplorer/widgets/importwizard.py

```diff
@@ -236,7 +238,14 @@ def get_row_sep(self):
 
     def get_skiprows(self):
         """Return number of lines to be skipped"""
-        return int(to_text_string(self.skiprows_edt.text()))
+        skip_rows = to_text_string(self.skiprows_edt.text())
+        # QIntValidator does not handle '+' sign
+        # See Spyder PR #20070
```

Comment:
Let change this to:
```suggestion
        # See spyder-ide/spyder#20070
```
To follow the comments link format supported by Spyder